docs/readme.md

```diff
@@ -1,23 +1,42 @@
-# Project Overview
+# Getting started
 
-The Gutenberg project is transforming the way content is created on WordPress. A block editor was the first product launched creating a new methodology for working with content.
+"Gutenberg" is the codename for the new WordPress editor, also known as the block editor. The Gutenberg project is transforming the way content is created on WordPress.
```

Comment:
Gutenberg is not the codename for the   WordPress  editor for me. It's the code name for  the whole  project which is a bit more than that.  (Editor, FSE, and in the future, collaborative editing and  multi-lingual). Maybe I'm just nitpicking, I don't know

Reply:
Yes, you are right here. Looking at the README here https://github.com/WordPress/gutenberg/blob/master/README.md

""Gutenberg" is a codename for a whole new paradigm in WordPress site building and publishing, that aims to revolutionize the entire publishing experience as much as Gutenberg did the printed word. Right now, the project is in the first phase of a four-phase process that will touch every piece of WordPress -- Editing, Customization, Collaboration, and Multilingual -- and is focused on a new editing experience, the block editor."

I'll instead rephrase this part.

Reply:
I think it's fair to say that right now  we're in the second phase. I guess the readme is outdated as well.

Reply:
New: "Gutenberg" is the codename for the new WordPress editor, also known as the block editor. The Gutenberg project is transforming the way content is created on WordPress.

--->

Should we cut down on the text?

"Gutenberg" is the codename for the new WordPress editor, also known as the block editor. (Skipping sentence two?)

--

If including sentence 2. Should we adjust "on WordPress" to "in WordPress" ?